src/sentry/incidents/endpoints/project_alert_rule_index.py

```diff
@@ -1,17 +1,72 @@
 from __future__ import absolute_import
 
+import math
+from datetime import datetime
 from copy import deepcopy
 
 from rest_framework import status
 from rest_framework.response import Response
 
+from django.utils import timezone
+
 from sentry import features
 from sentry.api.bases.project import ProjectEndpoint
 from sentry.api.exceptions import ResourceDoesNotExist
 from sentry.api.paginator import OffsetPaginator
-from sentry.api.serializers import serialize
+from sentry.api.serializers import serialize, AlertRuleSerializer, CombinedRuleSerializer
 from sentry.incidents.models import AlertRule
-from sentry.incidents.endpoints.serializers import AlertRuleSerializer
+from sentry.models import Rule, RuleStatus
+from sentry.utils.cursors import build_cursor, Cursor
+
+
+class ProjectCombinedRuleIndexEndpoint(ProjectEndpoint):
+    def get(self, request, project):
+        """
+        Fetches alert rules and legacy rules for an organization
+        """
+        if not features.has("organizations:incidents", project.organization, actor=request.user):
+            raise ResourceDoesNotExist
+
+        cursor_string = request.GET.get("cursor", None)
+        page_size = 25
```

Comment:
We should probably accept `page_size` as a querystring param, similar to how other paginators handle it (including enforcing a max value)

Reply:
Sure thing. What is our standard limit, and response to that limit being exceeded? Should I spit out a response with an error message like "limit exceeded", or just set the limit to the max and return that many results?

Reply:
I'd say just limit the max and return that many results, no need to throw an error